cmd/neofs-adm/internal/modules/morph/container.go

```diff
@@ -22,6 +22,37 @@ import (
 
 var errInvalidContainerResponse = errors.New("invalid response from container contract")
 
+func getContractHash(cmd *cobra.Command, inv *invoker.Invoker, c Client) (util.Uint160, error) {
```

Comment:
Can we name this `getContainerContractHash` at least?

Reply:
Renamed.